### ISSUE TITLE
Changes needed for index.html and style.css file @rexyagami

### ISSUE THREAD

**Issue by tushar-collab** [incubateind/gdtc-Day2#73]

**Comment by rexyagami** [comment]
@tushar-collab can you elaborate, please?

**Comment by tushar-collab** [comment]
1. padding in sections 2. aligning the heading in the center 3. and labels
in the progress bar are incorrect

**Comment by rexyagami** [comment]
@tushar-collab yes, that's correct you have to change and resolve the issues and design flaws to achieve the final design as shown in the image.
